### ISSUE TITLE
Implement functionality to set up a Night-Mode for the app

### ISSUE THREAD

**Issue by stefanieschreiner** [amosproj/amos2022ws02-automotive-test-app#50]
## User story
1. As a User
2. I want to be able to set up a Night-Mode in the App
3. So that I have the possibility to use a color scheme of light text & symbols on a dark background 

With this User story the functionality of setting up Night-Mode in the whole app is to be done. 
Night-Mode should stay activated, until it’s actively deactivated
If the Night-Mode is activated, it should be activated for the whole app, also when switching screens.

## Acceptance criteria
* The functionality of setting up Night-Mode is implemented.
* Night-Mode stays activated until its deactivated
* If Night-Mode is activated, it should apply to the whole app.  

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
